examples/fabric-port-connection/versions.tf

```diff
@@ -4,11 +4,14 @@ terraform {
   required_providers {
     equinix = {
       source  = "equinix/equinix"
-      version = ">= 1.7.0"
+      version = "~> 1.14"
     }
     google = {
       source  = "hashicorp/google"
       version = ">= 3.72.0"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-gcp/fabric-port-connection"
+  }
```

Comment:
Examples don't need to define provider_meta, it's just for the module itself
```suggestion
```

Reply:
updated